pkg/apis/nmstate/v1alpha1/state_types.go

```diff
@@ -0,0 +1,25 @@
+package v1alpha1
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
```

Comment:
Would you mind dropping this? It looks like boilerplate docs for bootstrapping. Noise!

Reply:
ack